examples/morpheusvm/go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/ava-labs/hypersdk/examples/morpheusvm
 
-go 1.21.10
```

Comment:
This is because of the CVE patched in `go 1.21.11`?

Reply:
This was done by updating AvalancheGo dependency since it bumps it.

I don't think the CVE impacts HyperSDK, so this is in accordance with updating to the latest patch version from one minor release behind.